src/model/schema.js

```diff
@@ -383,6 +383,34 @@ export default class Schema {
 	}
 
 	/**
+	 * Returns the lowest limit element defined in {@link module:engine/model/schema~Schema#limits} for passed selection.
```

Comment:
Returns the lowest {@link module:engine/model/schema~Schema#limits limit element} containing the entire selection or the root otherwise.